icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java

```diff
@@ -642,6 +644,81 @@ public void unitMeasure() {
                 ULocale.forLanguageTag("es-MX"),
                 1,
                 "kelvin");
+
+        // TODO(icu-units#35): skeleton generation.
+        assertFormatSingle(
+                "Mixed unit",
+                null,
+                "unit/yard-and-foot-and-inch",
+                NumberFormatter.with()
+                        .unit(MeasureUnit.forIdentifier("yard-and-foot-and-inch")),
+                new ULocale("en-US"),
+                3.65,
+                "3 yd, 1 ft, 11.4 in");
+
+        // TODO(icu-units#35): skeleton generation.
+        assertFormatSingle(
+                "Mixed unit, Scientific",
+                null,
+                "unit/yard-and-foot-and-inch E0",
+                NumberFormatter.with()
+                        .unit(MeasureUnit.forIdentifier("yard-and-foot-and-inch"))
+                        .notation(Notation.scientific()),
+        new ULocale("en-US"),
+                3.65,
+                "3 yd, 1 ft, 1.14E1 in");
+
```

Comment:
Indentation looks wrong.  Should be 8 spaces in from the function, so 16 spaces total

Reply:
done